variablecompany/bite
# Restaurant Detail View

## Describe your changes

This PR adds the restaurant detailed view of the selected restaurant either from home or if it is recognized from a NFC tag

## How to test
- Open the app
- Click on a restaurant
- check that the view renders correctly

## Checklist before requesting a review
